README.md

```diff
@@ -31,6 +31,17 @@ Sealed Secrets are *write only*, and as such, cannot be read directly from the d
 
 **Note**: Due to logistical issues with retrieving it on demand, the public certificate is stored in this repository under `k8s/production/sealed-secrets/cert.pem`. This is the *public* part of the public/private key pair, and is **not** sensitive information. The secrets scripts will use this certificate automatically, but if there is ever a need to use a *different* certificate, it can be set with the `SEALED_SECRETS_CERT` environment variable.
 
+### Fetching the private key
+There are some situations where you need to fetch the private key from the public/private key pair, in order to decode a sealed secret. In this case, the key can be fetched by running the following command:
+
+```
+kubectl get secret -n kube-system sealed-secrets-key-pair -o json | jq -r '.data."tls.key"' | base64 --decode > private.key
```

Comment:
Not a big deal, but I believe we can eliminate the dependence on `jq` here by just using the native `jsonpath` output format -
```suggestion
kubectl get secret -n kube-system sealed-secrets-key-pair -o jsonpath '{.data.tls\.key}' | base64 --decode > private.key
```